### ISSUE TITLE
Instructions for MEDIATOR for .env outdated

### ISSUE THREAD

**Issue by xtrycatchx** [openwallet-foundation/bifold-wallet#86]
I might be wrong, but is the README outdated ? - the value of the MEDIATOR should be the invitation url

```sh
MEDIATOR_URL=https://somewhere.mediatorx.io?c_i=ewogICJAdHlwZSI6ICJodHRwczovL2RpZGNvbW0ub3JnL2Nvbm5lY3Rpb25zLzEuMC9pbnZpdGF0aW9uIiwKICAiQGlkIjogIjE1NTZmMzZjLTRjZmUtNGQ1OS05YTliLTNmNmQ0MWFiZmVkMCIsCiAgImxhYmVsIjogIkF3ZXNvbWUuTWVkaWF0b3IiLAogICJyZWNpcGllbnRLZXlzIjogWwogICAgIjltdmVoZ3BzckNrNWcxdWlWNDNVQUhOSkVjemdCZEVMVHdEZXk2U3ZXRkgyIgogIF0sCiAgInNlcnZpY2VFbmRwb2ludCI6ICJodHRwczovL3NvbWV3aGVyZS5tZWRpYXRvcnguaW8iCn0=
GENESIS_URL=http://......./genesis
```

**Comment by JamesKEbert** [comment]
Hi @xtrycatchx! Yes, you'd be correct, I've updated the docs in https://github.com/hyperledger/aries-mobile-agent-react-native/pull/81
I've resolved some requested changes from Karim, so I'd expect that to be merged here soon!

**Comment by JamesKEbert** [comment]
Fixed in #81
